build-system/tasks/prepend-global/index.js

```diff
@@ -228,7 +275,8 @@ async function removeConfig(target) {
     return;
   }
   sanityCheck(contents);
-  const config = /self\.AMP_CONFIG\|\|\(self\.AMP_CONFIG=.*?\/\*AMP_CONFIG\*\//;
+  const config =
+    /self\.AMP_CONFIG\|\|\(self\.AMP_CONFIG=(.|\n)*?\/\*AMP_CONFIG\*\//;
```

Comment:
To prevent bugs from future maintainers, can we add a one-liner explainer comment for this regex?

Reply:
Sure! I believe this regex is simply for removing the AMP_CONFIG from the start of the file

Reply:
Actually, I don't think a 1-liner will help with this one since it is helpfully within a self-documenting fn named `removeConfig`, so I think the purpose is relatively clear.

The actual mechanism could use some documenting. I'll do that in a quick followup PR, as there are certain other changes I'd like to try to make before EOD that I don't want to block on!